### DISCUSSION TITLE
give me advice about the caption system in kohya_ss.

### DISCUSSION THREAD

**Discussion by hyunghoon-kim** [bmaltais/kohya_ss#2978] [Q&A]
When training SDXL,
I have a question about the captioning system in kohya_ss.

If there is no caption file (e.g., .txt file) and the folder name is 1_ohwx man,
does “ohwx man” get used as the caption for the text encoder?

On the other hand, if there is a .txt file,
does “ohwx man” not get used as the caption, and instead, the contents of the .txt file are used as the caption?

If anyone knows, I’d appreciate your advice.

(A person struggling with kohya_ss’s captioning system)

**Comment by Erlandsson** [comment]
what i understand is that if no caption texts, the lora becomes generic so to speak. Some say the filename will be the caption, some say it will not.
My first loras were without any captions, and did not have any problems using them.
But captions help when you want to gett better results.

I dont use blip or blip2 at all, its pretty useless captions from it. so generic and short ones.. 
WD14 is MUCH better in my opinion.
